calculator.py

```diff
@@ -0,0 +1,39 @@
+import sys
+
+try:
+    symbol, x, y = sys.argv[1], sys.argv[2], sys.argv[3]
+except:
+    print('error')
+    sys.exit()
+
+if (not x.isdigit()) or (not y.isdigit()):
+    print('error')
```

Comment:
It is better to choose one type of quotes for the whole code. 

> This PEP does not make a recommendation for this. Pick a rule and stick to it. 
https://peps.python.org/pep-0008/#string-quotes

Please, look at this PEP8 doc bellow